src/openrct2-ui/windows/Options.cpp

```diff
@@ -88,6 +88,7 @@ enum WINDOW_OPTIONS_WIDGET_IDX {
     WIDX_MULTITHREADING_CHECKBOX,
     WIDX_USE_VSYNC_CHECKBOX,
     WIDX_MINIMIZE_FOCUS_LOSS,
+    WIDX_ENABLE_SCREEN_SAVER,
```

Comment:
Your string says "disable" but the widget uses "enable" internally. This should be made consistent throughout.

Reply:
I will change the name to disable_screensaver_lock, as this seems clearer to me.
What do you think about that?